### ISSUE TITLE
Channel Grouping

### ISSUE THREAD

**Issue by horknfbr** [ZyC0R3/RitaBot#271]
**Is your feature request related to a problem? Please describe.**
it would be great to have a way to group channels together to allow for easier setup of cross translations.

for example, there are 3 main channels [foo, bar, baz]. and you support 3 languages [EN, JP, ES].  the current solution is to have foo-en -> foo-jp, foo-en -> foo-es, foo-jp -> foo-en, foo-jp -> foo-es, etc.... or the actual commands ```!t channel from English to Japanese for #foo-jp``` etc...

**Describe the solution you'd like**
give the ability to create a "channel group" and set the language for each channel and then the cross translation could be done without creating a mesh of channels.  

example: (in this example cg = channel group)
```
!t cg create foocg
!t cg add #foo-en to foocg
!t cg add #foo-jp  to foocg
!t cg add #foo-es to foocg
!t cg set lang to English for #foo-en
!t cg set lang to Japanese for #foo-jp
!t cg set lang to Spanish for #foo-es
```

there would also need to be a way to easily display the groups and the members with their assigned language 

```
!t cg show foocg
!t cg show #foo-en
```

**Describe alternatives you've considered**
the alternative I have used is a script that generates all the commands to mesh the channels so they all cross translate, for my setup that was 220 commands....  and I only have 4 main channels with 9 supported languages, a large server would be orders of magnitude more commands to run.

**Additional context**
this is a feature used by nvu and smoogle, so this would make it easier for users to transition to an open-source project rather than relying on paid solutions.

**Comment by ArtanisTheOne** [comment]
This can be done by creating a 3rd table named **groups** in the database which would hold channelID, serverID, and channelLang

**Comment by horknfbr** [comment]
looks like this is a duplicate of #100

**Comment by ArtanisTheOne** [comment]
Ah yes, you should move your message above there to so it adds to the conversation of what we want 😁

**Comment by ZyC0R3** [comment]
Yes, this is the plan, but its never been at the top of the list over all the other cahnges we had to make, but now she is mostly stable we can start to look at this.
